tests/unit/how-to-run-tests.md

```diff
@@ -0,0 +1,11 @@
+How to run unit tests
+
+unit tests must follow the "test*.py" regex to be picked up
+
+1. Individual tests
+
```

Comment:
you put `\` in this for file paths and `/` in the next step. Please keep it consistent. I am partial to `/` as its the unix one(manraj, zain, brady, and I use unix based systems).